specificatie/openapi.yaml

```diff
@@ -159,6 +159,66 @@ paths:
         default:
           $ref: 'https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/responses/default'
 
+  #19, 306, 373 
+  /adressen/_zoek:
```

Comment:
de user stories stonden niet duidelijk en correct beschreven, blijkt nu. Er is geen behoefte aan contour zoeken op adressen (geeft @CathyDingemanse aan), alleen op adresseerbare objecten en panden. Deze kan dus verwijderd worden (scheelt weer want ik vermoed dat deze het meest ingewikkeld is, een adres heeft immers geen geometrie).

Reply:
OK